### PR TITLE
[KIECLOUD-328] required changes to build meta information for CPaaS

### PR DESCRIPTION
**Pull request by errantepiphany**
[KIECLOUD-328] required changes to build meta information for CPaaS
https://issues.jboss.org/browse/KIECLOUD-328

Signed-off-by: David Ward <dward@redhat.com>

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ version: "1.4"
 from: "openshift3/apb-base:v3.11"
 labels:
   - name: "com.redhat.component"
-    value: "rhpam-apb"
+    value: "rhpam-7-rhpam76-apb-openshift-apb"
```
Comment (ruromero) [inline image.yaml]: Shouldn't that be `rhpam-7-rhpam76-apb-openshift-container` ?

Comment (spolti) [reply image.yaml]: It is part of CPaaS project, David sent a email about it.

Comment (spolti) [reply image.yaml]: ignore my comment :+1:

Comment (errantepiphany) [reply image.yaml]: @ruromero , please see my comment here:

https://projects.engineering.redhat.com/browse/RCM-65840?focusedCommentId=1847345&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1847345

I will also put it in the email thread. @spolti : FYI

**Pull request merged**